galaxy_ng/app/settings.py

```diff
@@ -100,3 +100,7 @@
     "COMPONENT_SPLIT_REQUEST": True,
     "dynaconf_merge": True,
 }
+
+# Disable django guardian anonymous user
+# https://django-guardian.readthedocs.io/en/stable/configuration.html#anonymous-user-name
+ANONYMOUS_USER_NAME = None
```

Comment:
Seems like the right fix.  Kind of wonder if it should somewhere up the stack (pulpcore?)